modules/op-configuring-pipelines-as-code-for-a-github-app.adoc

```diff
@@ -0,0 +1,78 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-pipelines-as-code-for-a-github-app_{context}"]
+= Configuring {pac} for a GitHub App 
+
+[role="_abstract"]
+GitHub Apps act as a point of integration with {pipelines-title} and bring the advantage of Git-based workflows to Tekton Pipelines. Cluster administrators can configure a single GitHub App for all users of the cluster. For GitHub Apps to work with Pipelines as Code, ensure that the webhook of the GitHub App points to the Pipelines as Code controller route (or ingress endpoint) that listens for GitHub events.
+
+[id="configuring-github-app-for-pac"]
+== Configuring a GitHub App
+
+[discrete]
+.Procedure
```

Comment:
this is the 2nd way of doing.. a little tougher
all this steps are automated using the 
`tkn pac bootstrap` cmd
if we can mention that first and then this that would be helpful
wdyt?
cc @chmouel